_data/wildfly-categories.yaml

```diff
@@ -0,0 +1,140 @@
+---
+categories:
+  - name: Agroal
+    id: agroal
+    description: Agroal Data Source
+  - name: Batch
+    id: batch
+    description: Jakarta EE Batch
+  - name: Bean Validation
+    id: bean-validation
+    description: Jakarta EE Bean Validation
+  - name: Bootable Jar
+    id: bootable-jar
+    description: Bootable Jar
+  - name: Command Line Interface (CLI)
+    id: cli
+    description: Command Line Interface
+  - name: Cloud
+    id: cloud
+    description: Developing and Running a WildFly application on the cloud
+  - name: User Experience
+    id: user-experience
+    description: Impact how the user experiences working with WildFly
+  - name: Clustering
+    id: clustering
+    description: Clustering, including Infinispan and JGroups
+  - name: Concurrency
+    id: concurrency
+    description: Concurrency
+  - name: Console
+    id: console
+    description: Console
+  - name: WildFly Controller
+    id: controller
+    description: WildFly Controller
+  - name: WildFly Core
+    id: core
+    description: WildFly Core
+  - name: Jakarta EE
+    id: ee
+    description: Jakarta EE
+  - name: Jakarta EJB
+    id: ejb
+    description: Jakarta Entreprise Java Beans (EJB)
```

Comment:
Jakarta Enterprise Beans (EJB)

Reply:
Note you have a typo too -- 'Entre'

Reply:
That one is alway tripping me ("Entreprise" is a French word)